src/pyhf/infer/calculators.py

```diff
@@ -168,3 +169,160 @@ def _false_case():
                 (sqrtqmu_v < self.sqrtqmuA_v), _true_case, _false_case
             )
         return teststat
+
+
+class EmpiricalDistribution(object):
+    """
+    The empirical distribution of the test statistic.
+
+    Unlike ~pyhf.infer.calculators.AsymptoticTestStatDistribution where the
```

Comment:
I'm not sure why, but if I add `calculators.EmpiricalDistribution` to the `api.rst` in docs this rendering doesn't come out correctly.
![EmpiricalDistribution_rendering_error](https://user-images.githubusercontent.com/5142394/77954047-ac052a00-7293-11ea-8d54-38ea98191520.png)

Reply:
Resolved with 
```
:py:class:`target-goes-here`
```